.ruby-version

```diff
@@ -1 +0,0 @@
-2.1.3
```

Comment:
Also this one is fairly important (at least for me on my mac) it says which ruby version to use.